vgen/src/main/java/io/gapi/vgen/py/PythonLanguageProvider.java

```diff
@@ -58,9 +58,10 @@
 
     Doc filenameDoc = snippets.generateFilename(element);
     String outputFilename = filenameDoc.prettyPrint();
+    Doc body = snippets.generateBody(element);
     List<String> importList = importHandler.calculateImports();
     // Generate result.
-    Doc result = snippets.generateClass(element, importList);
+    Doc result = snippets.generateModule(element, body, importList);
```

Comment:
So the Python codegen (before your change) figures out all of the imports in an independent process from generating the class; it appears that you are converting it to work like the Java one now, where imports are accumulated as part of generating the body (in a stateful way). I don't have much opinion on whether one of these approaches should be used universally, but I wanted to call out this change in approach, because the authors of Python codegen may have picked their approach for a particular reason. @ethanbao @geigerj